lib/Runtime/Library/ArrayBuffer.h

```diff
@@ -325,7 +326,8 @@ namespace Js
         // take over ownership. a CoTaskMemAlloc'ed buffer passed in via projection.
         static ProjectionArrayBuffer* Create(byte* buffer, DECLSPEC_GUARD_OVERFLOW uint32 length, DynamicType * type);
         virtual void Dispose(bool isShutdown) override;
-        virtual void Finalize(bool isShutdown) override {};
+        virtual void Finalize(bool isShutdown) override;
+        virtual bool IsProjectionArrayBuffer() override { return true; }
```

Comment:
>IsProjectionArrayBuffer [](start = 21, length = 23)

Trying to find out where is this is used ?

Reply:
This is no longer needed. It was use in the iteration where we special cased projection buffers.

---
In reply to: [186509650](https://github.com/Microsoft/ChakraCore/pull/5099#discussion_r186509650) [](ancestors = 186509650)